lxd/db/generate/db/parse.go

```diff
@@ -139,12 +139,48 @@ func Parse(pkg *ast.Package, name string, kind string) (*Mapping, error) {
 		return nil, errors.Wrapf(err, "Failed to parse %q", name)
 	}
 
+	filterStr := findStruct(pkg.Scope, name+"Filter")
```

Comment:
@masnax just a general comment on code style here. When generating a variable for use in a function (so in this case `name+"Filter"` which you then go onto use in the returned error's context. I would suggest assigning that to a separate variable and then using it in both the function arg and the error. That way if it changes in the future you don't have to update it into multiple places and the value doesn't have to be generated twice.